<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoDialog</class>
 <widget class="QDialog" name="TodoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QFrame" name="editFrame">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QTextEdit" name="descriptionEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="summaryEdit">
          <property name="placeholderText">
           <string>summary</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="selectFrame">
       <layout class="QGridLayout" name="selectLayout">
        <item row="0" column="0">
         <widget class="QComboBox" name="todoListSelector">
          <property name="toolTip">
           <string>select your todo list</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="todoList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
